### ISSUE TITLE
Can you explain why you made changes in DisparityRegression?

### ISSUE THREAD

**Issue by MaitaYuki** [SpadeLiu/Graft-PSMNet#11]
@SpadeLiu ,

I observed that you added the following codes in DisparityRegression compared with the original function in PSMNet. Can you explain the purpose?
###########
        disp = torch.arange(0, self.max_disp).view(1, -1, 1, 1).float().to(x.device)

        if self.win_size > 0:
            max_d = torch.argmax(x, dim=1, keepdim=True)
            d_value = []
            prob_value = []
            for d in range(-self.win_size, self.win_size + 1):
                index = max_d + d
                index[index < 0] = 0
                index[index > x.shape[1] - 1] = x.shape[1] - 1
                d_value.append(index)

                prob = torch.gather(x, dim=1, index=index)
                prob_value.append(prob)

            part_x = torch.cat(prob_value, dim=1)
            part_x = part_x / (torch.sum(part_x, dim=1, keepdim=True) + 1e-8)
            part_d = torch.cat(d_value, dim=1).float()
            out = torch.sum(part_x * part_d, dim=1)
##########
